src/me/drton/jmavsim/MAVLinkDisplayOnly.java

```diff
@@ -0,0 +1,77 @@
+package me.drton.jmavsim;
+
+import me.drton.jmavlib.conversion.RotationConversion;
+import me.drton.jmavlib.mavlink.MAVLinkMessage;
+import me.drton.jmavlib.mavlink.MAVLinkSchema;
+import me.drton.jmavsim.vehicle.AbstractVehicle;
+
+import javax.vecmath.*;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * MAVLinkDisplayOnly is MAVLink bridge between AbstractVehicle and autopilot connected via MAVLink.
+ * MAVLinkDisplayOnly should have the same sysID as the autopilot, but different componentId.
+ * It reads HIL_STATE_QUATERNION from the MAVLink and displays the vehicle position and attitude.
+ * @author Romain Chiappinelli
+ */
+public class MAVLinkDisplayOnly extends MAVLinkHILSystem {
```

Comment:
Since this overrides/ignores most functionality of MAVLinkHILSystem, I'd rather have this inherit from `MAVLinkSystem` instead.
And as `Simulator` uses part of the `MAVLinkHILSystem` interface, we can introduce a `MAVLinkHILSystemBase` between `MAVLinkSystem` and `MAVLinkHILSystem`, which declares the interface required for `Simulator`.
What do you think?

Reply:
That makes sense, I will think about it.

Reply:
I am checking the feasibility, it becomes difficult as `Visualizer3D` needs `MAVLinkHILSystem`, same for `hilSystem.gotHilActuatorControls()`. 
1. We can have one variable of type `MAVLinkHILSystemBase`, instantiate `MAVLinkDisplayOnly` or `MAVLinkHILSystem` and cast them when needed:
`MAVLinkHILSystemBase hilSystem`
`((MAVLinkHILSystem)hilSystem).gotHilActuatorControls()`
2. We can have two distinct objects, one `MAVLinkHILSystem` and one `MAVLinkDisplayOnly`

I think the second option is preferable. What do you think ?

Reply:
The third option is to add a `gotHilActuatorControls()` to `MAVLinkHILSystemBase`, and override it as needed. I see that as the cleanest way, or am I missing something?